pkg/templates/templates.go

```diff
@@ -78,3 +78,13 @@ func (t *Templates) GetDockerfile() ([]byte, error) {
 	tmpl := fmt.Sprintf("templates/build/%s.Dockerfile", t.Data.BuildDriver)
 	return t.fs.ReadFile(tmpl)
 }
+
+// GetCredentialSetJSON returns a credential-set.schema.json template file to define new credential set.
+func (t *Templates) GetCredentialSetJSON() ([]byte, error) {
+	return t.fs.ReadFile("templates/credentials/create/credential-set.schema.json")
```

Comment:
There is a bit of an inconsistency here between what the command does when json is requested (returning a schema) instead of yaml (returning a usable file with placeholders). The original issue was looking for the latter behavior, placeholders.

The schema for credential sets is stored in https://github.com/getporter/porter/blob/release/v1/pkg/schema/credential-set.schema.json and we point people to it from this documentation page: https://release-v1.porter.sh/reference/file-formats/